__tests__/app.test.js

```diff
@@ -163,6 +163,63 @@ describe("app", () => {
             });
         });
       });
+      describe("GET /articles/:article_id/comments", () =>{
+        test("200: returns an object containing an array", () => {
+          return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then(({ body }) => {
+              expect(Array.isArray(body.comments)).toBe(true);
+            });
+        });
+        test("200: returns the array of comments for the correct article with all standard comment properties", () => {
```

Comment:
Minor tweaks made to this test since pushing, see https://github.com/tjejack/be-nc-news/commit/18e7fb416e7946bc768dabe93849750b64ea8c45